SabreTools.DatItems/DatItem.cs

```diff
@@ -207,6 +207,7 @@ public static DatItem Create(BaseFile baseFile)
                     return new Media(baseFile);
 
                 case FileType.CHD:
+                case FileType.ISOArchive:
```

Comment:
`Disk` has a very specific meaning. `Disk` refers exclusively to CHDs that use the `disk` tag in the various DAT formats. Overloading this with `ISO` as well is not expected nor is it something I want to support in this manner. If you wish to create a new tag for ISOs, please do that instead.

Reply:
see later, transform into chd is what the basic idea is, so I used Disk for it. Transform into ISO is not used yet...